pkg/clusterapi/identity_test.go

```diff
@@ -105,7 +105,7 @@ func TestConfigureAWSIAMAuthInKubeadmControlPlane(t *testing.T) {
 								},
 							},
 							ControllerManager: bootstrapv1.ControlPlaneComponent{
-								ExtraArgs: map[string]string{},
+								ExtraArgs: clusterapi.SecureTlsCipherSuitesExtraArgs(),
```

Comment:
same here about not using `clusterapi.SecureTlsCipherSuitesExtraArgs()`
we can have a method in the tests like `defaultControllerManagerTests()` so you can reuse that in all these tests if you want